client/settings/views/wcc-settings-form/settings-item.js

```diff
@@ -102,20 +100,15 @@ const SettingsItem = ( {
 			);
 
 		case 'packages':
-			const packagesState = form.packages;
+			const packagesMsg = sprintf(
+				__( 'Manage your packages on <a href="%(url)s">the packages settings page</a>' ),
+				{
+					url: '/wp-admin/admin.php?page=wc-settings&tab=connect&section=packages',
+				}
+			);
+
 			return (
-				<Packages
-					{ ...packagesState }
-					{ ...packagesActions }
-					packages={ fieldValue }
-					presets={ schema.definitions.preset_boxes }
-					dimensionUnit={ storeOptions.dimension_unit }
-					removePackage={ removeArrayItem }
-					savePackage={ savePackage }
-					weightUnit={ storeOptions.weight_unit }
-					errors={ errors }
-					packageSchema={ fieldSchema.items }
-				/>
+				<div dangerouslySetInnerHTML={ { __html: packagesMsg } } />
```

Comment:
I'm not a fan of using `dangerouslySetInnerHTML` when there's not really needed. What about:
```
<div>
    { __( 'Manage your packages on' ) } <a href="/wp-admin...">{ __( 'the package settings page' ) }</a>
</div>
```

This would make it weirder for the translators, so I don't have a strong opinion about it. Your call :)

Also, I would add `target="_blank"` to the link.